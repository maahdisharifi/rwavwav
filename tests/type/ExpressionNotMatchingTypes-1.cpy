int main():
    float x = 3.14
    float y = x + "123"
    return 0
end
