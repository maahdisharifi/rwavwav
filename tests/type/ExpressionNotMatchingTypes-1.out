Line:3 -> type mismatch in expression